hieradata_aws/class/cache.yaml

```
@@ -15,3 +15,7 @@ govuk::apps::router::mongodb_nodes:
   - 'router-backend-3'
 
 varnish::environment_ip_prefix: "%{hiera('environment_ip_prefix')}"
+
+# Increase the maximum number of file descriptors usable by the router
+# A file descriptor is used per connection
+govuk::deploy::config: 65536
```

Does this need to specify the parameter?

```suggestion
govuk::deploy::config::nofile_limit: 65536
```

---

Indeed, good spot :+1: